misc/languages.json

```diff
@@ -1 +1 @@
HUSK","v":"ISO-8859-1","asm-x64":"ascii","teascript":"latin-1"},"tio":{"js":"javascript-babel-node","java":"java-openjdk","java9":"java-openjdk9","cs":"cs-csc","fs":"fs-mono","c":"c-gcc","cpp":"cpp-gcc","sed":"sed-gnu","pf":"pain-flak","tf":0,"matlab":0,"jolf":0,"vba":0,"vb":"visual-basic-net-vbc","brachylog":"brachylog2","retina":"retina1","cubix":"cubix","qbasic":0,"smilebasic":0,"excel":0,"sql":0,"asm-x64":"assembly-gcc","teascript":0,"vim":0},"cm":{"js":"javascript","java":"text/x-java","java9":"text/x-java","haskell":1,"powershell":1,"python2":"python","python3":"python","bash":"text/x-sh","perl5":"perl","perl6":"perl","c":"text/x-csrc","cpp":"text/x-c++src","matlab":"octave","japt":"javascript","r":1,"ruby":2,"mathematica":1,"cs":"text/x-csharp","brainfuck":2,"fs":"text/x-fsharp","apl-dyalog":"text/apl","apl-ngn":"apl","arble":"lua","vba":"text/x-vb","vb":2,"php":2,"groovy":2,"vsl":"text/x-swift","sql":2,"scala":2,"asm-x64":"text/x-gas"},"alias":{"javascript":"js","apl":"apl-dyalog","osabie":"05ab1e","c++":"cpp","befunge":"befunge-98","perl":"Perl 5","bf":"brainfuck"},"highlight":{"bash":"sh","c":1,"cpp":1,"java":1,"java9":"java","perl5":"perl","perl6":"perl","js":1,"powershell":1,"mathematica":1,"haskell":1,"matlab":"matlab","japt":"js","r":1,"ruby":1,"roda":"rodacode","arble":"lua","vba":"vbnet","vb":"vbnet","php":1,"groovy":1,"qbasic":"basic","vsl":"swift","clean":1,"smilebasic":"basic","excel":1,"sql":1,"scala":1,"asm-x64":"x86asm","vim":1},"languages":{"05ab1e":{"display":"05AB1E"},"actually":{},"apl-dyalog":{"display":"APL (Dyalog)"},"apl-ngn":{"display":"APL (ngn)"},"bash":{},"befunge-98":{},"brainfuck":{"display":"Brainf***","sn":"BF"},"c":{},"cheddar":{},"clean":{},"cpp":{"display":"C++","sn":"C+"},"cs":{"display":"C#","sn":"C#"},"emojicode":{},"fs":{"display":"F#","sn":"F#"},"funky":{},"gaia":{},"husk":{},"java":{},"java9":{"display":"Java 9"},"jelly":{},"js":{"display":"JavaScript"},"haskell":{},"perl5":{"display":"Perl 5"},"perl6":{"display":"Perl 6"},"python2":{"display":"Python 2"},"python3":{"display":"Python 3"},"pyke":{},"pyth":{},"sed":{"display":"GNU Sed"},"seriously":{},"powershell":{"display":"PowerShell"},"mathematica":{},"stacked":{},"triangularity":{},"addpp":{"display":"Add++"},"lolcode":{"display":"LOLCODE","sn":"LC"},"element":{},"pf":{"display":"Pain-Flak"},"tf":{"display":"Tf","sn":"Tf"},"matlab":{"display":"MATLAB","sn":"ML"},"japt":{"display":"Japt"},"jolf":{},"matl":{"display":"MATL"},"r":{},"ruby":{},"roda":{"display":"Röda","sn":"RÖ"},"arble":{"display":"ARBLE"},"vba":{"display":"VBA"},"vb":{"display":"VB.NET"},"php":{"display":"PHP"},"charcoal":{},"brachylog":{"sn":"BL"},"retina":{},"v":{},"cjam":{"display":"CJam"},"cubix":{},"j":{},"groovy":{},"brain-flak":{},"qbasic":{"display":"QBasic"},"vsl":{"display":"VSL"},"smilebasic":{"display":"SmileBASIC"},"excel":{},"sql":{"display":"SQL"},"scala":{},"asm-x64":{"display":"ASM (x64)","sn":"AS"},"racket":{},"teascript":{"display":"TeaScript","sn":"TS"},"vim":{}}}
```

Comment:
In the future, could we format this as one k-v pair per line so that it's easier to read the diff?